giraffa-core/src/main/java/org/apache/giraffa/GiraffaConstants.java

```diff
@@ -19,10 +19,12 @@
 
 public interface GiraffaConstants {
```

Comment:
The signatures of the enums and fields in this class do not need the `public static final` modifier.